app/javascript/admin/controllers/data_update_script_controller.js

```diff
@@ -33,10 +33,13 @@ export default class DataUpdateScriptController extends Controller {
       if (response.ok) {
         this.pollForScriptResponse(id, statusColumn, runAtColumn);
       } else {
+        const fileNameElement = document.getElementById(
```

Comment:
The reason I set the variable in both places is because it didn't make sense to declare the variable upfront and pass it around when we weren't sure that we're going to use it. It only gets used in the case of an error and when the counter runs out.